cmd/argo/commands/retry.go

```diff
@@ -37,5 +51,6 @@ func NewRetryCommand() *cobra.Command {
 	command.Flags().StringVarP(&cliSubmitOpts.output, "output", "o", "", "Output format. One of: name|json|yaml|wide")
 	command.Flags().BoolVarP(&cliSubmitOpts.wait, "wait", "w", false, "wait for the workflow to complete")
 	command.Flags().BoolVar(&cliSubmitOpts.watch, "watch", false, "watch the workflow until it completes")
+	command.Flags().StringVar(&retryOps.nodesToResetFieldSelector, "reset-nodes-field-selector", "", "selector of nodes to reset, eg: --node-field-selector inputs.paramaters.myparam.value=abc")
```

Comment:
I think the name of this flag should be `--node-field-selector` as all this does is provide a selector. This way it would be analogous to #1904. To specify that we want said nodes restarted we can pass a flag in conjunction:

```
$ argo retry --restart-successful --node-field-selector inputs.paramaters.myparam.value=abc
```
Or something like this. What do you think?

Reply:
Good idea, am applying.